### PR TITLE
[FSSDK-10950] cleanup of test files

### PR DESCRIPTION
**Pull request by raju-opti**
## Issues
- FSSDK-10950

### CODE REVIEW

File: lib/project_config/project_config_manager.spec.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ describe('ProjectConfigManagerImpl', () => {
     expect(logger.error).toHaveBeenCalled();
   });
 
-  it('should set status to Failed if neither datafile nor a datafileManager is passed into the constructor', async () => {
+  it.only('should set status to Failed if neither datafile nor a datafileManager is passed into the constructor', async () => {
```
Comment (junaed-optimizely) [inline lib/project_config/project_config_manager.spec.ts:37]: Please remove this `only`.

File: lib/project_config/project_config_manager.ts [hunk 2]
```diff
@@ -73,7 +73,7 @@ export class ProjectConfigManagerImpl extends BaseService implements ProjectConf
 
     if(this.isSsr) {
       // If isSsr is true, we don't need to poll for datafile updates 
-      this.datafileManager = undefined 
+      // this.datafileManager = undefined
```
Comment (junaed-optimizely) [inline lib/project_config/project_config_manager.ts:76]: Is this supposed to be commented out?

**Review by junaed-optimizely — CHANGES_REQUESTED** [review]

File: lib/project_config/project_config_manager.spec.ts (hunk 1 above)
Comment (raju-opti) [reply lib/project_config/project_config_manager.spec.ts]: updated

File: lib/project_config/project_config_manager.ts (hunk 2 above)
Comment (raju-opti) [reply lib/project_config/project_config_manager.ts]: no. updated

**Review by junaed-optimizely — APPROVED** [review]
LGTM!

**Pull request merged**